packages/ifpack/src/Ifpack_Hypre.h

```diff
@@ -326,6 +339,18 @@ class Ifpack_Hypre: public Ifpack_Preconditioner {
    */
     int SetParameter(Hypre_Chooser chooser, int (*pt2Func)(HYPRE_Solver, int*), int* parameter);
 
+    //! Set a parameter that takes an int**.
+    /*!
+    \param chooser (In) -A Hypre_Chooser enumerated type set to Solver or Preconditioner, whatever the parameter is setting for.
```

Comment:
I'm not sure if this is correct Doxygen syntax, but it's consistent with what Ifpack does.